project-base/storefront/components/Pages/Customer/complaintFormMeta.ts

```diff
@@ -24,7 +24,7 @@ export const useComplaintForm = (
     const resolver = yupResolver(
         Yup.object().shape<Record<keyof ComplaintFormType, any>>({
             quantity: Yup.string()
-                .matches(/^[0-9]*$/, t('Please enter quantity'))
+                .matches(/^[1-9]*$/, t('Please enter quantity'))
```

Comment:
I don't think this will work, you won't be able to set quantity 10 for example. Let's try `^[1-9][0-9]*` instead.